jax/_src/third_party/scipy/linalg.py

```diff
@@ -110,3 +112,74 @@ def funm(A: ArrayLike, func: Callable[[Array], Array],
           tol, (tol / minden) * norm(jnp.triu(T, 1), 1))))
 
   return F, err
+
+
+@jit
+def solve_sylvester(A: jnp.ndarray, B: jnp.ndarray, C: jnp.ndarray) -> jnp.ndarray:
+    """
+    Solves the Sylvester equation:
+      .. math:: AX + XB = C
+
+    Parameters
```

Comment:
Please use the docstring style that matches the rest of the file (`Args:` rather than `Parameters`, no underline on sections, parameter descriptions without types on their own line, etc.)